randovania/games/samus_returns/gui/preset_settings/msr_hints_tab.py

```diff
@@ -1,30 +1,38 @@
+from __future__ import annotations
+
 import dataclasses
+from typing import TYPE_CHECKING
 
 from PySide6 import QtCore
 
-from randovania.game_description.game_description import GameDescription
 from randovania.games.samus_returns.gui.generated.preset_msr_hints_ui import Ui_PresetMSRHints
 from randovania.games.samus_returns.layout.hint_configuration import ItemHintMode
 from randovania.games.samus_returns.layout.msr_configuration import MSRConfiguration
 from randovania.gui.lib.signal_handling import set_combo_with_value
-from randovania.gui.lib.window_manager import WindowManager
 from randovania.gui.preset_settings.preset_tab import PresetTab
-from randovania.interface_common.preset_editor import PresetEditor
-from randovania.layout.preset import Preset
+
+if TYPE_CHECKING:
+    from randovania.game_description.game_description import GameDescription
+    from randovania.gui.lib.window_manager import WindowManager
+    from randovania.interface_common.preset_editor import PresetEditor
+    from randovania.layout.preset import Preset
 
 
 class PresetMSRHints(PresetTab, Ui_PresetMSRHints):
-    def __init__(self, editor: PresetEditor, game_description: GameDescription, window_manager: WindowManager):
+    def __init__(self, editor: PresetEditor, game_description: GameDescription, window_manager: WindowManager) -> None:
         super().__init__(editor, game_description, window_manager)
         self.setupUi(self)
 
         self.hint_layout.setAlignment(QtCore.Qt.AlignmentFlag.AlignTop)
 
         for i, item_hint_mode in enumerate(ItemHintMode):
             self.hint_artifact_combo.setItemData(i, item_hint_mode)
-
         self.hint_artifact_combo.currentIndexChanged.connect(self._on_art_combo_changed)
 
+        for i, baby_hint_mode in enumerate(ItemHintMode):
```

Comment:
That's the same iteration as in L28. Could be combined for less code but at the same time it's cleaner if every combo box has its own block. 🤔